boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -56,34 +58,36 @@ class FsmStateMachineTest : public testing::Test
             get_current_ms, HEARTBEAT_MONITOR_TIMEOUT_PERIOD_MS,
             HEARTBEAT_MONITOR_BOARDS_TO_CHECK, heartbeat_timeout_callback);
 
-        primary_flow_meter_in_range_check = App_InRangeCheck_Create(
+        primary_flow_rate_in_range_check = App_InRangeCheck_Create(
```

Comment:
Unrelated to this PR but I'm changing these names to stay consistent with the `InRangeCheck` structs in `App_FsmWorld.c`